app/controllers/api/v1x0/portfolios_controller.rb

```diff
@@ -2,25 +2,29 @@ module Api
   module V1x0
     class PortfoliosController < ApplicationController
       include Api::V1x0::Mixins::IndexMixin
+      include Api::V1x0::Mixins::RBACMixin
```

Comment:
below `include Api::V1x0::Mixins::RBACMixin` 
we should have something like: https://guides.rubyonrails.org/action_controller_overview.html#filters
```
before_action :write_access_check, :only => [:add_portfolio_item_to_portfolio, :create, :update, :destroy]
```
Then we can remove the duplicated calls